### PR TITLE
Change collections.Collection to collections.abc.Collection

### PR DESCRIPTION
**Pull request by martmists**

### CODE REVIEW

File: gyukutai/__init__.py [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import random
 import typing
 import types
-import collections
+import collections.abc
 # If you ever see this in a project, run.
 # This is never a good thing.
 import forbiddenfruit
 
+if "Collection" in collections.abc.__all__:
```
Comment (SunDwarf) [inline gyukutai/__init__.py]: why is this not a hasattr() check

Comment (martmists) [inline gyukutai/__init__.py]: totally fogot about hasattr, will fix

**Comment by SunDwarf** [comment]
test in production

**Comment by SunDwarf** [comment]
although using this in production is a mistake

**Comment by martmists** [comment]
using this is a mistake in the first place

**Pull request merged**